config/server.json

```diff
@@ -8,5 +8,9 @@
   },
   "mongo": {
     "connectionString": "mongodb://localhost/streams"
+  },
+  "schemaVersion": {
+    "gte": 0,
```

Comment:
Might be clearer if it was "minimum" and "maximum" (rather than "gte" and "lte"). Just my 2 cents, though...